orca-web/src/main/groovy/com/netflix/spinnaker/orca/data/jackson/StageMixins.groovy

```diff
@@ -1,25 +0,0 @@
-/*
- * Copyright 2014 Netflix, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.netflix.spinnaker.orca.data.jackson
-
-import com.fasterxml.jackson.annotation.JsonIgnore
-import com.netflix.spinnaker.orca.pipeline.model.Pipeline
-
-interface StageMixins {
-  @JsonIgnore
-  Pipeline getPipeline()
-}
```

Comment:
I think that's handled by the `@JsonBackReference` on `Stage.execution` and it's interesting there was apparently never an equivalent for `Orchestration`.